### PR TITLE
Add the order book panel component.

### PR DESCRIPTION
**Pull request by biocom**
Adding a general orderBookPanel component, which looks like this right now:
Screen Shot 2020-11-20 at 14 58 32

And eventually should look like:
Screen Shot 2020-11-20 at 14 58 54

### CODE REVIEW

**Comment by biocom** [comment]
Will make use of our Tabs component eventually.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr1639--dexreact.review.gnosisdev.com)**: Testing web app
   * **📚 [Storybook](https://pr1639--dexreact.review.gnosisdev.com/storybook/)**: Component stories

File: src/components/OrderBookPanel/index.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+//     title: 'SELL',
+//     content: '- sell component -',
+//     activeColor: '--color-sell',
+//   },
+// ]
+
+const Wrapper = styled.div`
+  display: flex;
+  padding: var(--padding-container-default);
+  background: none;
```
Comment (Velenir) [inline src/components/OrderBookPanel/index.tsx:29]: Why?I don't see it inheriting background

Comment (biocom) [reply src/components/OrderBookPanel/index.tsx:29]: I guess defensive CSS syntax?

**Pull request closed without merging**